dashboard/src/locales/messages/index.ts

```diff
@@ -185,8 +185,8 @@ export const messages = {
     'testStatus.skip': 'Skip',
     'testsTab.errorsSummary': 'Errors Summary',
     'testsTab.fail': 'Fails',
+    'testsTab.hardwareErrors': 'Hardware with errors',
```

Comment:
there is still testDetails.platform in this file, you can search and it goest to test details

Reply:
ditto about these are just hardware tested now